cloudify_docker_image/README.md

```diff
@@ -24,4 +25,27 @@
 	  export DOCKER_ID_USER="username"
 	  export DOCKER_ID_PASSWORD="password"
 	  ```
-  - You can setup your own tags for the image by adding more items to the array `IMAGE_TAGS`.
\ No newline at end of file
+  - You can setup your own tags for the image by manipulating the array `IMAGE_TAGS`.
+- Pull image from the repository (may require a `docker login`)
+  ```
+  # Pull the image
+  docker pull cloudifycosmo/<REPO_NAME>:<TAG>
+  
+  # Spin up the manager
+  docker run --name <ARBITRARY_CONTAINER_NAME> -d -v /sys/fs/cgroup:/sys/fs/cgroup:ro --tmpfs /run \
+    --tmpfs /run/lock --security-opt seccomp:unconfined --cap-add SYS_ADMIN cloudifycosmo/<REPO_NAME>:<TAG>
+  
+  # Get the IP address
+  CONTAINER_IP=$(docker inspect --format='{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' <ARBITRARY_CONTAINER_NAME>)
+  
+  # Now you can use $CONTAINER_IP to connect to the manager
```

Comment:
Do you mean connect using cloudify CLI? Otherwise, why do I need the IP?

Reply:
Yes to connect to the Manager via CLI. I thought it would be important to add how to get the container's IP. Should I keep it?

Reply:
Yeah, keep it, let's just change it to something like "to connect to the manager via the CLI"

Reply:
Fixed. Can I merge?